src/main/java/lotto/rule/LottoWinningRule.java

```diff
@@ -0,0 +1,39 @@
+package lotto.rule;
+
+import java.util.Arrays;
+
+public enum LottoWinningRule {
+    RANK_ONE(6, false, 2000000000, "6개 일치 (2,000,000,000원) - %d개"),
```

Comment:
이렇게 숫자의 자리수가 많아질 때는 `_` 구분자로 자리수를 표현할 수 있답니다!
```suggestion
    RANK_ONE(6, false, 2_000_000_000, "6개 일치 (2,000,000,000원) - %d개"),
```

Reply:
오 처음 알았네요! 감사합니다😊